README.md

```diff
@@ -210,6 +210,8 @@ s.string.array.lengthGt(5); // Must have more than 5 elements
 s.string.array.lengthGe(5); // Must have 5 or more elements
 s.string.array.lengthEq(5); // Must have exactly 5 elements
 s.string.array.lengthNe(5); // Must not have exactly 5 elements
+s.string.array.lengthRange(0, 4); // Must have at least 0 elements and less than 4 elements (in math, that is [0, 4))
```

Comment:
Missing closing ] here

Reply:
If I'm not wrong then `[0,4)` means including 0 but not 4.

Reply:
I am not missing a closing `]` at all! Math range of `[0, 4)` means 0, 1, 2, 3